runtime/Makefile

```diff
@@ -37,7 +37,7 @@ install: containerd-shim-aws-firecracker
 	install -D -o root -g root -m755 -t $(INSTALLROOT)/bin containerd-shim-aws-firecracker
 
 test:
-	go test ./... $(EXTRAGOARGS)
+	DISABLE_ROOT_TESTS=true go test ./... $(EXTRAGOARGS)
```

Comment:
Why did you make this change?

Reply:
The new test that was added, TestBuildJailedRootHandler, uses `chown` which requires root.

Reply:
So why did you disable them in the Makefile?

Reply:
Cause I followed what we did for the snapshotter stuff. If we want to do something different, let me know

Reply:
Do these tests get run by the `integ-test` target now?

Reply:
No, no root tests are run in our CI system from the looks of it.

Reply:
How do you plan for the tests to be run then?

Reply:
No idea, but we have other root tests that aren't being run. So we need to figure out a solution. To run as root we'd need to do this, https://github.com/buildkite/elastic-ci-stack-for-aws/issues/330. Are we okay adding that to sudo configuration? Seems a little too open to me.

Reply:
We've generally been trying to run tests that require elevated privileges inside Docker containers, unless my memory is failing me.  @sipsma, is that the way you had set it up?

Reply:
Root tests are configured to run in our CI system here: https://github.com/firecracker-microvm/firecracker-containerd/blob/fcaf03274de5118fad7fd1a245b40580a353abed/.buildkite/pipeline.yml#L56-L76

All tests in the runtime package that require root access or other forms of isolation just need to be suffixed with `_Isolated` and use `internal.RequiresIsolation(t)`, [such as here](https://github.com/firecracker-microvm/firecracker-containerd/blob/fcaf03274de5118fad7fd1a245b40580a353abed/runtime/service_integ_test.go#L93-L94). They will then be run in their own docker container.

Reply:
Ah cool! Fixed. Also I'll have another PR that has the other root test to make use of `_Isolated`